JitsiConferenceEvents.js

```diff
@@ -177,6 +177,17 @@ export const RECORDER_STATE_CHANGED = 'conference.recorderStateChanged';
 export const VIDEO_SIP_GW_AVAILABILITY_CHANGED
     = 'conference.videoSIPGWAvailabilityChanged';
 
+/**
+ * Indicates that video SIP GW Session state changed.
+ * @param {options} event - {
```

Comment:
nit: it looks like you were trying to document the event properties as an object but you didn't close the object.